framework/src/st_things/things_stack/easy-setup/easysetup_manager.c

```diff
@@ -456,7 +456,7 @@ static void *wifi_prov_set_loop(void *param)
 	THINGS_LOG_D(TAG, "Open Loop Enter.");
 
 	static int timeout_handle_num = 100;
-	things_timeout_s timeout;
+	static things_timeout_s timeout;
```

Comment:
could give more concrete info?  timeout is stack var,  who will free it beforeused in new threads as param?

Reply:
The reference of this stack object `timeout` is passed to new thread at line 495, 
and all stack objects will be freed after current function returned at line 501. 
In the new thread `timeout` dereference may happen later, but `timeout` stack object
does not exist any more at that time.

Reply:
As disscussed offline, this object will be cloned when creating new thread, there's no problem!
I reverted the change, thanks for your review!